deku-p/src/core/protocol/protocol.ml

```diff
@@ -110,6 +112,7 @@ let apply_operation ~current_level protocol operation :
               with
               | vm_state, ledger -> Some (ledger, Some receipt, vm_state, None)
               | exception External_vm_client.Vm_execution_error error ->
+                  let () = failwith error in
```

Comment:
This is a mistake from my end, forgot to drop it before sending the patch

Reply:
no worries, I'll remove it